.github/workflows/doc-build.yml

```diff
@@ -63,3 +63,43 @@ jobs:
       with:
         name: htmldocs.tar
         path: htmldocs.tar
+
+  doc-build-pdf:
+    name: "Documentation Build (PDF)"
+    runs-on: ubuntu-latest
+    container: texlive/texlive:latest
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: install-pkgs
+      run: |
+        apt-get update
+        apt-get install -y python3-pip cmake doxygen librsvg2-bin
+
+    - name: cache-pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: pip-${{ hashFiles('scripts/requirements-doc.txt') }}
+
+    - name: install-pip
+      run: |
+        pip3 install -U setuptools wheel pip west
+        pip3 install -r scripts/requirements-doc.txt
+
+    - name: west setup
+      run: |
+        west init -l .
+
+    - name: build-docs
+      run: |
+        cmake -Sdoc -Bbuild -DSPHINXOPTS="-q -j auto" -DLATEXMKOPTS="-quiet -halt-on-error"
+        cmake --build build --target pdf
+
+    - name: upload-build
+      uses: actions/upload-artifact@master
+      continue-on-error: True
+      with:
+        path: build/latex/zephyr.pdf
```

Comment:
add name here, so it is visible that this is the PDF.

```
name: pdf-output
```

for example.

Reply:
added